### ISSUE TITLE
Set up proselint

### ISSUE THREAD

**Issue by underyx** [coala-analyzer/coala#2284]

**Comment by tushar-rishav** [comment]
I like this tool. But IMO this issue should rather go with coala-bears.
@sils1297 We want this?

**Comment by AbdealiJK** [comment]
We already have ProseLintBear - https://github.com/coala-analyzer/coala-bears/blob/master/bears/natural_language/ProseLintBear.py

We could look into improving the configs if you like, but it **is** there.
@underyx Is that what you meant by set-up ?

**Comment by underyx** [comment]
Yes, that's what I meant, sorry about the ambiguity. We had lots of
documentation commits lately and thought it would be nice to have writing
style checks for them.

**Comment by cylussec** [comment]
I would be willing to take this on, but its not clear what you are looking for. Should there be some automated process to look at documentation changes and use Proselint on them automatically? Or are you looking for someone to go through the existing documentation?

**Comment by sils1297** [comment]
@underyx ?

**Comment by sils** [comment]
I think it's basically adding ProseLintBear to the coafile of the coala/documentation repository so our docs are checked, @underyx can you confirm?

**Comment by underyx** [comment]
Yep, that's it.

**Comment by d1ndra** [comment]
Would like to work on this if nobody isn't.

**Comment by sils** [comment]
Hi! Welcome to our community and thank you, both @cylussec and @d1ndra for your interest!

I have sent you an invitation to join our community. This is needed so we can coordinate tasks with you and make sure no two persons are working on the same thing. If you can't find the mail, you should be able to join us at http://github.com/coala-analyzer/!

> Hint: if you're proud to be part of the team, you can make your membership public at https://github.com/orgs/coala/people

Rules as follows, since both of you want to do it: the first one to accept the invitation and confirming this on this issue will get it. I'm happy to help the other one to find another low difficulty issue.

We'd love to be able to help you when problems occur. Just join us on [our gitter channel](https://coala.io/chat) and tell us about them :)

**Comment by jayvdb** [comment]
As noted on https://github.com/coala/documentation/pull/244, there are false positives due to https://github.com/amperser/proselint not understanding code blocks in RST.  It seems that upstream is interested, but someone needs to write the bug description, and probably do some leg work to make it happen.

An alternative is to use LanguageTool.  I've raised that as an issue at https://github.com/coala/documentation/issues/283 .
